packages/components/src/tooltip/stories/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../';
+import Button from '../../button';
+
+export default {
+	title: 'Components/ToolTip',
+	component: Tooltip,
+};
+
+export const _default = () => {
+	const tooltipText = text( 'Text', 'More information' );
+	return (
+		<Tooltip text={ tooltipText }>
+			<Button isSecondary>Hover for more information</Button>
+		</Tooltip>
+	);
+};
```

Comment:
Do you think it would be good to show multiple variations on the same story (top, left, right, bottom) like we do for the "buttons" story. Or we could use a knob for the position.

Reply:
Yeah, I started to do that but because storybook renders without any padding, changing the position didn't have any effect. I can add a wrapper element that positions the tooltip in the center, so you could see the difference?

Reply:
Sounds like a good plan to me.